### DISCUSSION TITLE
Welcome to PyroEdgeGPTBot Discussions!

### DISCUSSION THREAD

**Discussion by tom-snow** [tom-snow/PyroEdgeGPTBot#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TechyCSR** [comment]
[FeatureRequest]
Currently supports only the allowed users , can it be used for public use 
in short anybody can use it .

**Reply by tom-snow** [reply]
did v2 version works well?

**Reply by TechyCSR** [reply]
I tried the V2 .
Use one daily interval we are getting upstream error

**Reply by TechyCSR** [reply]
Even before this version images were not getting error but after the update in EdgeGPt we are often getting redirect error .
No matter cookie updated or not

**Reply by TechyCSR** [reply]
Even time interval of getting response is increased

**Reply by TechyCSR** [reply]
It's all upstream error 
So I don't find good to disturb You .

Although You provided great code , thank You so much

**Reply by tom-snow** [reply]
![image](https://github.com/tom-snow/PyroEdgeGPTBot/assets/79245287/1910b28c-01df-4ef4-a125-db306ca1856b)


it works well for me. I deploy it on railway.

Just /cookie command first then upload a cookie.json file.

**Reply by tom-snow** [reply]
and /update command may help

**Comment by TechyCSR** [comment]
I got You

**Comment by TechyCSR** [comment]
But I have userbase of 8000 people every day

**Comment by TechyCSR** [comment]
That got me problem na

**Comment by TechyCSR** [comment]
Even I tried to explain how to ensure Your personal Cookie file 
But don't worked yet

**Reply by TechyCSR** [reply]
People are using it without their Own Cookie
Also API limit hitting

**Reply by tom-snow** [reply]
you can try prepare lots of cookie files and edit the code to random choose a cookie to use. A lot of bing accounts need indeed

**Reply by TechyCSR** [reply]
I thought the same and even doing that .

**Reply by TechyCSR** [reply]
But You only thinks when You are getting 8000 + user's and 700+ image generation request every day 

Even the text responses are getting 500 + question every hour 

How we can work ?

**Reply by TechyCSR** [reply]
See here's my log 

![Screenshot_2023-05-25-10-43-04-299_org telegram messenger](https://github.com/tom-snow/PyroEdgeGPTBot/assets/90786270/9cca5a96-9efe-422a-b4b6-7f3909b8ad79)
![Screenshot_2023-05-25-10-42-55-522_org telegram messenger](https://github.com/tom-snow/PyroEdgeGPTBot/assets/90786270/2b071f05-8102-4b5f-9f45-aec8118d00d3)
![Uploading Screenshot_2023-05-25-10-42-58-598_org.telegram.messenger.jpg…]()

**Reply by TechyCSR** [reply]
I created my log channel to ensure the eye over the bot system
Above are it's screenshots

**Reply by Yorddyyorddy** [reply]
oh no users privacy gone🫠

**Reply by TechyCSR** [reply]
Nahh , we removed that log channel after the same issue from ppl 🥲.

**Reply by Yorddyyorddy** [reply]
nop you can save the error logs only or count per request ,
can I view your bot?

**Reply by TechyCSR** [reply]
Sure You can view 
Soon I am going to launch an advance version of it 
Here's current version
Username 
@AI_INDIA_bot 

You can checkout :)

**Reply by Yorddyyorddy** [reply]
got error 
anyways I have personal tg bot only built in for myself

**Reply by TechyCSR** [reply]
Alright 
Error on what?

**Reply by Yorddyyorddy** [reply]
> Alright Error on what?

saving to dict and retrieving later

**Reply by TechyCSR** [reply]
Okays :)

**Reply by Yorddyyorddy** [reply]
so where's ?
@CSR-SOFTWARES
